### PR TITLE
fixed CQ for bitmessageqt.languagebox module

### PR DESCRIPTION
**Pull request by coolguy-cell**
pylint fixes for bitmessageqt.languagebox module

### CODE REVIEW

File: src/bitmessageqt/languagebox.py [hunk 1]
```diff
@@ -8,24 +10,27 @@
...
+        """Populate comboBox based on languageName"""
         self.clear()
         localesPath = os.path.join(paths.codePath(), 'translations')
         self.addItem(QtGui.QApplication.translate(
             "settingsDialog", "System Settings", "system"), "system")
         self.setCurrentIndex(0)
         self.setInsertPolicy(QtGui.QComboBox.InsertAlphabetically)
         for translationFile in sorted(
-            glob.glob(os.path.join(localesPath, "bitmessage_*.qm"))
+                glob.glob(os.path.join(localesPath, "bitmessage_*.qm"))
```
Comment (g1itch) [inline src/bitmessageqt/languagebox.py:33]: not necessary

File: src/bitmessageqt/languagebox.py [hunk 2]
```diff
@@ -8,24 +10,27 @@
...
 
 class LanguageBox(QtGui.QComboBox):
+    """LanguageBox class for QT UI"""
     languageName = {
         "system": "System Settings", "eo": "Esperanto",
         "en_pirate": "Pirate English"
     }
 
     def __init__(self, parent=None):
+        """Initialization"""
```
Comment (g1itch) [inline src/bitmessageqt/languagebox.py:20]: not necessary, not informative

File: src/bitmessageqt/languagebox.py [hunk 3]
```diff
@@ -8,24 +10,27 @@
 
 
 class LanguageBox(QtGui.QComboBox):
+    """LanguageBox class for QT UI"""
```
Comment (g1itch) [inline src/bitmessageqt/languagebox.py:13]: Qt is not QT

Comment (coolguy-cell) [reply src/bitmessageqt/languagebox.py:13]: @g1itch sorry for that give me a minute,I am fixing it right now.

File: src/bitmessageqt/languagebox.py (hunk 1 above)
Comment (coolguy-cell) [reply src/bitmessageqt/languagebox.py:33]: @g1itch pylint complains for it 

C: 33, 0: Wrong hanging indentation before block (add 4 spaces).
            glob.glob(os.path.join(localesPath, "bitmessage_*.qm"))
            ^   | (bad-continuation)

File: src/bitmessageqt/languagebox.py [hunk 4]
```diff
@@ -8,24 +10,27 @@
...
         "en_pirate": "Pirate English"
     }
 
     def __init__(self, parent=None):
+        """Initialization"""
         super(QtGui.QComboBox, self).__init__(parent)
         self.populate()
 
     def populate(self):
+        """Populate comboBox based on languageName"""
```
Comment (g1itch) [inline src/bitmessageqt/languagebox.py:25]: Wrong. It populates drop down list with all available languages.

File: src/bitmessageqt/languagebox.py (hunk 1 above)
Comment (g1itch) [reply src/bitmessageqt/languagebox.py:33]: It's a bug: https://github.com/psf/black/issues/48

File: src/bitmessageqt/languagebox.py (hunk 2 above)
Comment (coolguy-cell) [reply src/bitmessageqt/languagebox.py]: updated.

File: src/bitmessageqt/languagebox.py (hunk 3 above)
Comment (coolguy-cell) [reply src/bitmessageqt/languagebox.py]: fixed.

File: src/bitmessageqt/languagebox.py (hunk 4 above)
Comment (coolguy-cell) [reply src/bitmessageqt/languagebox.py]: @g1itch done.

File: src/bitmessageqt/languagebox.py (hunk 1 above)
Comment (g1itch) [reply src/bitmessageqt/languagebox.py:32]: I tired to point all of you to the same bugs and PEP8 articles...

Comment (coolguy-cell) [reply src/bitmessageqt/languagebox.py:32]: @g1itch so for now I should undo this ?

Comment (PeterSurda) [reply src/bitmessageqt/languagebox.py:32]: undo change and override pylint

Comment (coolguy-cell) [reply src/bitmessageqt/languagebox.py]: Done.

**Pull request merged**